packages/shared/utils/sort-nav-items.js

```diff
@@ -0,0 +1,5 @@
+module.exports = items => items.sort((a, b) => {
+  if (a.label > b.label) return 1;
+  if (a.label < b.label) return -1;
+  return 0;
+});
```

Comment:
Locale comparison would be best when sorting strings for differences in case, etc.
```suggestion
module.exports = items => items.sort((a, b) => `${a.label}`.localeCompare(b.label));
```